include/vcmi/FactionMember.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * FactionMember.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+#include "Entity.h"
+
+VCMI_LIB_NAMESPACE_BEGIN
+
+class BonusList;
+
+namespace PrimarySkill
+{
+    enum PrimarySkill : int8_t;
+}
+
+class DLL_LINKAGE AFactionMember: public IConstBonusProvider, public INativeTerrainProvider
+{
+public:
+	/**
+	 Returns native terrain considering some terrain bonuses.
+	*/
```

Comment:
I am for `/* */` for all comments except commented code blocks. If it is principial for you - can change, comment style is not so principial for me. What do you think?

Reply:
Personally prefer `///` for documentation - since with such format comment is always on the line immediately before method (without empty line between method & comment).

Not insisting on change but I guess I wish we could decide on common style.